main.py

```diff
@@ -16,6 +16,10 @@ def run(provider_info: tuple) -> None:
             volume_size_remaining = 100 - org_volume_sizes["percent"]
             log.info(f"VOLUME Size  ::  {volume_size_remaining}% Remaining..")
 
+            if envs.SPACE_LEFT_ALERT == True:

```

Comment:
In python, you do not need to explicitly evaluate a bool (or similar - "" , [], {} ...)

You just use
```
if envs.SPACE_LEFT_ALERT:
    ...
```

  For False you would use _**not**_

 ```
  if not envs.SPACE_LEFT_ALERT:
    ... 
```